AutomatedTesting/Gem/PythonTests/Prefab/tests/instantiate_prefab/InstantiatePrefab_WithNestedEntities.py

```diff
@@ -0,0 +1,75 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+def InstantiatePrefab_WithNestedEntities():
+    """
+    Test description:
+    - Creates linear nested entities.
+    - Creates a prefab from the nested entities.
+    - Instantiates another copy of the prefab and validates the structure is intact.
+
+    Test passes if the 3 entities inside the newly instanced prefab have correct hierarchy and positions.
+    """
+
+    from pathlib import Path
+
+    import azlmbr.legacy.general as general
+    import azlmbr.math as math
+
+    from editor_python_test_tools.prefab_utils import Prefab
+    from editor_python_test_tools.wait_utils import PrefabWaiter
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    import Prefab.tests.PrefabTestUtils as prefab_test_utils
+
+    NESTED_ENTITIES_PREFAB_FILE_NAME = Path(__file__).stem + 'nested_entities_prefab'
+    NESTED_ENTITIES_NAME_PREFIX = 'Entity_'
+    POSITION = math.Vector3(100.0, 100.0, 100.0)
+    NUM_NESTED_ENTITIES_LEVELS = 3
+
+    prefab_test_utils.open_base_tests_level()
+
+    # Creates new nested entities at the root level
+    # Asserts if creation didn't succeed
+    nested_entities_root = prefab_test_utils.create_linear_nested_entities(NESTED_ENTITIES_NAME_PREFIX,
+                                                                           NUM_NESTED_ENTITIES_LEVELS, POSITION)
```

Comment:
stylistically you could newline after the parenthesis in each of these lines to alter the readability, this reduces the dangling parameters on lines by themselves. its really a style trade off choice as long as you consistently go one way or the other it wouldn't matter. spacing in this suggestion may not be accurate to what you would do.
```suggestion
    nested_entities_root = prefab_test_utils.create_linear_nested_entities(
                               NESTED_ENTITIES_NAME_PREFIX, NUM_NESTED_ENTITIES_LEVELS, POSITION)
```